docs/source/conf.py

```diff
@@ -163,19 +163,19 @@
 ]
 
 # -- Extension configuration -------------------------------------------------
-# autoapi_dirs = [
-#     '../../src',
-# ]
-# autoapi_add_toctree_entry = False
-# autoapi_options = [
-#     'members',
-#     'undoc-members',
-#     'private-members',
-#     'show-inheritance',
-#     'show-module-summary',
-#     'special-members',
-#     #    'imported-members',
-# ]
+autoapi_dirs = [
```

Comment:
To get `autoapi` working, you'll also need to uncomment line 51 of this file to add it to the list of extensions.